float64[4] joints 
geometry_msgs/Pose goal_pose